src/components/MapsContainer.ts

```diff
@@ -154,11 +162,12 @@ class MapsContainer extends Component<MapsContainerProps, MapsContainerState> {
                         mxform: this.props.mxform,
                         nanoflow: locationOptions.dataSourceNanoflow,
                         contextObject,
-                        inputParameterEntity: locationOptions.inputParameterEntity
+                        inputParameterEntity: locationOptions.inputParameterEntity,
+                        requiresContext
                     })
                     .then(mxObjects => this.setLocationsFromMxObjects(mxObjects, locationOptions))
                     .then(locations => resolve(locations))
-                    .catch(reason => reject(`${reason}`));
+                    .catch(reason => reject(reason.message));
```

Comment:
Add a type for `reason` if possible? And maybe it should now be called `error` instead of reason.

Reply:
Refactored all promisses